src/icons/item-icon/ItemIcon.tsx

```diff
@@ -81,7 +81,16 @@ export interface ItemIconProps {
 
 const ItemIcon = ({ className, dimension = 32, iconType, title }: ItemIconProps) => {
     const IconComponent = (itemIconTable as { [key: string]: React.FC<SVGProps> })[iconType] || IconFileDefault;
-    return <IconComponent className={className} height={dimension} title={title} width={dimension} />;
+    return (
+        <IconComponent
+            className={className}
+            height={dimension}
+            title={title}
+            width={dimension}
+            aria-hidden="true"
+            role="presentation"
```

Comment:
Does the `ItemIcon` component always need to have these props or should they be passed in based on use?

Reply:
yes `ItemIcon` always need to have those properties

Reply:
alternative is to identify the type of file and then set an `alt` property based on that
but I guess is better to just hide it for screen reader users

Reply:
https://github.com/box/box-ui-elements/pull/2686